src/webgpu/shader/validation/parse/align.spec.ts

```diff
@@ -178,3 +178,62 @@ g.test('align_required_alignment')
 
     t.expectCompileResult(!fails, code);
   });
+
+g.test('align_non_struct')
+  .desc('Test that alignment is only allowed on struct members')
```

Comment:
This doesn't seem very exhaustive in terms of where the attribute can go. 

Would it be better to parameterize a list of shaders, with insertion points for attributes, then have a big table of `attribute + location = valid` that we can check? That way, you can validate nearly all the attributes for all locations.

Reply:
Done. Made it specific to align at the moment. Want to get a few more in place to see if/how it generalizes.